### PR TITLE
added index.html

### PR DESCRIPTION
**Pull request by amandagaabo**
I added an index.html file to the project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
```
Comment (amandaherschleb) [inline index.html]: add lang=en so it is accessible

File: index.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+<head>
+  <title>Learn GitHub</title>
+</head>
+<body>
+  <p>Just a throwaway index.html page.</p>
```
Comment (amandaherschleb) [inline index.html]: put an 'h1' element in here

File: index.html (hunk 1 above)
Comment (amandajezek) [inline index.html]: add lang=en to make this accessible

File: index.html (hunk 2 above)
Comment (amandajezek) [inline index.html]: add "h1" tag in the body

File: index.html (hunk 1 above)
Comment (amandajezek) [reply index.html]: ok, will do

**Pull request merged**